src/sentry/integrations/discord/integration.py

```diff
@@ -16,6 +17,7 @@
 from sentry.integrations.discord.client import DiscordClient
 from sentry.integrations.discord.commands import DiscordCommandManager
 from sentry.pipeline.views.base import PipelineView
+from sentry.shared_integrations.exceptions import IntegrationError
 from sentry.shared_integrations.exceptions.base import ApiError
 from sentry.utils.http import absolute_uri
 
```

Comment:
didn't catch this earlier but we usually use

`from logging import logger` 

followed by 

`logger = logging.getLogger(__name__)` to report logs associated with the right path/package.